services/fitter/fitter.py

```diff
@@ -11,7 +11,7 @@
 
 sys.path.append(".")
 from services.plotter.models import Configurations
-from services.plotter.plotter import graph_builder, get_data
+from services.plotter.plotter import graph_builder_plotter, get_data
```

Comment:
We should have an uniform way to define the graph builder. Focus on the proposing a good data structure